src/js/dismiss.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * DISMISS
+ *
+ * @package uri-component-library
+ */
+
+( function() {
+
+	'use strict';
+
+	window.addEventListener( 'load', init, false );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['init' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/uriweb/uri-component-library/pullRequest?prid=4536390)